ldes-server-admin/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/rest/controllers/DcatViewsRestController.java

```diff
@@ -13,6 +14,7 @@
 import static org.apache.jena.riot.WebContent.contentTypeNQuads;
 import static org.apache.jena.riot.WebContent.contentTypeTurtle;
 
+@Observed
```

Comment:
Do Http Calls not already get observed on their own?

Reply:
And the Http calls are also filtered out in the SimpleLoggingHandler, as the context from those calls are wrapped in a simple Observation.Context and we use the advanced ObservedAspect.ObservedAspectContext